plugins/metadata/dc11/ru_RU/locale.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale.xml
```

Comment:
Hi Alec, the en_US locale are not coherent when it comes to complete file names, we have quite a lot locale files with "locale.xml" (in OJS: 61 in total, 6 in en_US). Translators working as thoroughly as Pavel does, they follow what they find in the en_US files (I do the same). If you want to add complete paths to the locale files, let me know, I can take care of it. for the moment, I'll leave this at it is, because it is in sync with en_US.